crates/typst/src/layout/flow.rs

```diff
@@ -856,6 +960,44 @@ impl<'a, 'e> FlowLayouter<'a, 'e> {
 
         Ok(())
     }
+
+    fn layout_line_number(&mut self, numbering: &Numbering, y: Abs) -> SourceResult<()> {
+        let line_counter = Counter::of(ParLineMarker::elem());
```

Comment:
This is a general style nit and mostly just my personal preference, but (as also noted in the other PR), I think many variable names could be shortened. This makes the code a bit less verbose.

E.g.:
- `line_counter` => `counter` (we don't care about any other counter here, so it's clear enough)
- `line_counter_update` => either `update` or `counter_update`
- `line_number` => `number`